test/app/src/router/index.ts

```diff
@@ -1,20 +1,19 @@
-import Vue from 'vue';
-import VueRouter from 'vue-router';
-import { LoginCallback } from '@okta/okta-vue';
+import { createRouter, createWebHistory } from 'vue-router'
+import { LoginCallback, beforeNavigation, afterNavigation } from '@okta/okta-vue'
 import Protected from '@/components/Protected.vue';
 import SessionTokenLogin from '@/components/SessionTokenLogin.vue';
 
-Vue.use(VueRouter);
-
-const router = new VueRouter({
-  mode: 'history',
-  base: '/',
+const router = createRouter({
+  history: createWebHistory(process.env.BASE_URL),
   routes: [
     { path: '/implicit/callback', component: LoginCallback },
```

Comment:
I would suggest changing this to `/login/callback` since most of the other examples use this.

Reply:
It's the route in the e2e test app to handle the implicit flow. There is also one in the next line called `/pkce/callback`.

Reply:
It's the route in the e2e test app to handle the implicit flow. There is also one in the next line called /pkce/callback.